src/themes/Default.jsx

```diff
@@ -289,6 +289,9 @@ export const config = {
       backgroundColor: colors.red5,
       borderColor: colors.red5
     }
+  },
+  modal: {
+    backgroundColor: colors.black
```

Comment:
It's missing the color item ;)

Also, the default theme modal colors should be the same as the root colors of that theme. So the modal item should be:

```
  modal: {
    color: colors.anthracite1,
    backgroundColor: colors.white,
  }

```

Reply:
Given the fact that the `backgroundColor` of the modal is about the same color of the theme, but with opacity 90%, what you suggest makes the text unreadable.
I will leave `white` for now.

Reply:
No it's not ^^

The modal itself with anthracite text on white background is clearly readable. And the theme of the underneath app can/should have another theme than the modla theme.

Eg: A red app opens a default (white) modal

Furthermore, if the developer wants to use a dark modal, he can choose to use the Anthracite theme, but the default have to stay white as the other components behaviors.

Reply:
What about
```
    modal: {
      color: colors.black, // for default theme
      color: colors.black, // for light themes
      color: colors.white, // for dark themes
      backgroundColor: makeTransparent(color1, 0.9)
    }
```
?

Reply:
I don't understand why you don't want to apply the inplace themes ^^ I'm certainly missing a point but I can't figure it out.

All the themes have a background color and a text color and it's working well everywhere. I tested it on the Modal component and it's the same result.

So for the default theme it should be:
```
modal: {
  color: colors.anthracite1,
  backgroundColor: colors.white,
}
```

Reply:
For the dark one:
```
modal: {
  color: colors.white,
  backgroundColor: color2,
}
```

And for the light one:
```
modal: {
  color: dark27Color1,
  backgroundColor: color2,
}
```

Reply:
Ok, actually I DO want to apply the inplace themes, but I totally missed out what your color defaults were set up for each theme, and tried to figure out a 'ok' styling.
Sorry about that.

Reply:
Oh ok! ^^ No problem!

Reply:
(keeping in mind that I apply 90% transparency, which will change slightly the theme color)

Reply:
Yep